### PR TITLE
IGNITE-18330 Fix javadoc in Transaction#resume(), Transaction#suspend, CacheJdbcPojoStoreFactory#getBatchSize

### PR DESCRIPTION
**Pull request by y616468159**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [x] There is a single JIRA ticket related to the pull request. 
- [x] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/transactions/Transaction.java [hunk 1]
```diff
@@ -277,14 +277,14 @@ public interface Transaction extends AutoCloseable, IgniteAsyncSupport {
     public IgniteFuture<Void> rollbackAsync() throws IgniteException;
 
     /**
-     * Resume a transaction if it was previously suspended. <strong>Supported only for optimistic transactions.</strong>
+     * Resume a transaction if it was previously suspended. <strong>Now, It supports pessimistic transactions and optimistic transactions.</strong>
```
Comment (luchnikovbsk) [inline modules/core/src/main/java/org/apache/ignite/transactions/Transaction.java:280]: Might be better to delete - <strong>Supported only for optimistic transactions.</strong>?

File: modules/core/src/main/java/org/apache/ignite/transactions/Transaction.java [hunk 2]
```diff
@@ -277,14 +277,14 @@ public interface Transaction extends AutoCloseable, IgniteAsyncSupport {
...
      *
      * @throws IgniteException If resume failed.
      */
     public void resume() throws IgniteException;
 
     /**
-     * Suspends a transaction. It could be resumed later. <strong>Supported only for optimistic transactions.</strong>
+     * Suspends a transaction. It could be resumed later. <strong>Now, It supports pessimistic transactions and optimistic transactions.</strong>
```
Comment (luchnikovbsk) [inline modules/core/src/main/java/org/apache/ignite/transactions/Transaction.java:287]: Same as the previous comment.

**Review by luchnikovbsk — CHANGES_REQUESTED** [review]

File: modules/core/src/main/java/org/apache/ignite/transactions/Transaction.java (hunk 2 above)
Comment (y616468159) [reply modules/core/src/main/java/org/apache/ignite/transactions/Transaction.java]: Thank you for your guidance. Now I have resubmitted according to your suggestion.

Comment (y616468159) [reply modules/core/src/main/java/org/apache/ignite/transactions/Transaction.java]: Also, I submitted a [JIRA question](https://issues.apache.org/jira/browse/IGNITE-20024), but no one communicated with me. Should I submit the modifications directly?

**Comment by alex-plekhanov** [comment]
@y616468159, can you please also fix javadoc for `GridNearTxLocal#suspend` and `GridCacheSharedContext#suspendTx`? They contain notes about optimistic transactions too.

**Comment by y616468159** [comment]
> @y616468159，您能同时修复`GridNearTxLocal#suspend`和 的javadoc`GridCacheSharedContext#suspendTx`吗？它们还包含有关乐观交易的注释。



> @y616468159，您可以同时修复`GridNearTxLocal#suspend`和的javadoc`GridCacheSharedContext#suspendTx`吗？它们还包含有关乐观交易的注释。

**Comment by y616468159** [comment]
> @y616468159，您可以同时修复`GridNearTxLocal#suspend`和的javadoc`GridCacheSharedContext#suspendTx`吗？它们还包含有关乐观交易的注释。

Yes, I can solve it! May I submit it directly in the current pull request or do I need to create a separate pull request?

**Comment by alex-plekhanov** [comment]
@y616468159, it's better to use this pull request.

**Comment by y616468159** [comment]
@alex-plekhanov Fixed

**Review by alex-plekhanov — APPROVED** [review]

**Pull request closed without merging**